test/support/models.rb

```diff
@@ -128,6 +128,7 @@ def column_for_attribute(attribute)
 
   begin
     require 'active_model/type'
+    require 'active_record/type'
```

Comment:
Well, it worked without having to explicitly require it, but I'm removing the dependency on ActiveRecord anyway.